phpBB/phpbb/install/module_base.php

```diff
@@ -110,10 +110,38 @@ public function run()
 
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+		    // Receiving task index
```

Comment:
Could you explain what could lead to the increase in the execution time?

So you think that this implementation is incorrect? It seems to me that this is how it should be fixed. I think the user should be able to come back to the previous step.